be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -86,7 +76,13 @@ Status SinkBuffer::add_request(TransmitChunkInfo& request) {
     }
     {
         auto& instance_id = request.fragment_instance_id;
-        RETURN_IF_ERROR(_try_to_send_rpc(instance_id, [&]() { _buffers[instance_id.lo].push(request); }));
+        RETURN_IF_ERROR(_try_to_send_rpc(instance_id, [&]() {
+            auto& item = _instance_id_lo2item[instance_id.lo];
+#if !_GLIBCXX_USE_CXX11_ABI
```

Comment:
suggest to wrap a pair of std::size_t, std::list into a new type list, for ex example:

```cpp
template<typename T>
class SizedList {
public:
   push(...){
     _list.push(...);
#if !_GLIBCXX_USE_CXX11_ABI
     _size.fetch_add(1);
#endif
   }
....
private:
std::list<T> _list;
#if !_GLIBCXX_USE_CXX11_ABI
std::atomic<size_t> _size
#endif
}
```